dokka-integration-tests/gradle/build.gradle.kts

```diff
@@ -215,5 +219,25 @@ testing {
                 systemProperty("junit.jupiter.execution.parallel.mode.default", "CONCURRENT")
             }
         }
+
+        // Register specific Gradle test runs for each example so the examples can be tested individually.
+        exampleGradleProjectsDir.toPath()
+            .listDirectoryEntries()
+            .map { it.name }
+            .filter {
+                it.endsWith("-example")
+                // note: avoid checks like `isDirectory()` to prevent Gradle registering the files as CC inputs.
+            }
+            .forEach { exampleProjectName ->
+                val prettyName = exampleProjectName.split("-").joinToString("") { it.uppercaseFirstChar() }
+
+                targets.register("test$prettyName") {
```

Comment:
minor: it would be better to have names `testExampleXXX` instead of `testXXXExample` so that they are easier to find

Reply:
Ah, I missed that it's in a separate group - interesting :)
Thanks!